docs/csharp/language-reference/keywords/index.md

```diff
@@ -76,6 +76,15 @@ Keywords are predefined, reserved identifiers that have special meanings to the
 |[var](../../../csharp/language-reference/keywords/var.md)|[where (generic type constraint)](../../../csharp/language-reference/keywords/where-generic-type-constraint.md)|[where (query clause)](../../../csharp/language-reference/keywords/where-clause.md)|  
 |[yield](../../../csharp/language-reference/keywords/yield.md)||  
   
+## Single-character keywords
+
+Single-character keywords are contextual symbols that have a special meaning to the compiler. C# supports the following single-character keywords: 
+
+
+- The [@](../../../csharp/language-reference/keywords/verbatim.md) symbol is the verbatim identifier character. It can be used to distinguish an identifier (such as a variable named `@for`) from a C# keyword, or to indicate that the characters in a string are interpreted literally. 
+
+- The [$](../../../csharp/language-reference/keywords/interpolated.md)` symbol is the interpolated string character. It identifies literal string that follows as an interpolated string.
```

Comment:
stray backtick here

I'd also leave the explanation of what they do for the page reference since you don't explain what the other keywords do.